.github/workflows/cross.yaml

```diff
@@ -47,22 +50,29 @@ jobs:
       #   if: matrix.os == 'ubuntu-latest'
       #   run: sudo apt-get update && sudo apt-get install -yq libpcsclite-dev
       - name: build cosign
```

Comment:
maybe we can add `build cosign & sget` or something like that, but it is a non blocking